src/Core/View.js

```diff
@@ -201,6 +201,7 @@ class View extends THREE.EventDispatcher {
         this._pixelDepthBuffer = new Uint8Array(4);
 
         // Focus needed to capture some key events.
+        this.domElement.tabIndex = -1;
```

Comment:
Could you add comment (`tabIndex=-1`) inspired from https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Could you add comment to merge the PR?